textworld/envs/__init__.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT license.
 
-
+import textworld.envs.wrappers
```

Comment:
So that we can do:
```python
import textworld
textworld.render.wrappers.Recorder(...)
```
Without that new line, there is an error:
`AttributeError: module 'textworld.envs' has no attribute 'wrappers'`

Reply:
Ah okay.  To me that's just the behaviour I expect from Python modules.  If you want to use `a.b.c` you need to `import a.b.c` first, `import a` isn't enough.

If we have places in our code that do things like this I think they should be fixed to import what they really need.